conjure-java-core/src/integrationInput/java/com/palantir/product/SingleUnion.java

```diff
@@ -87,24 +87,29 @@ public String toString() {
 
         private Function<String, T> unknownVisitor;
 
+        @Override
```

Comment:
Mind adding a `final` modifier to the VisitorBuilder while we're here?

Reply:
Done